pyrail/irail.py

```
@@ -143,8 +143,10 @@ def _refill_tokens(self) -> None:
 
         # Refill tokens, 3 tokens per second, cap tokens at 3
         self.tokens = min(3, self.tokens + int(elapsed * 3))
-        # Refill burst tokens, 3 burst tokens per second, cap burst tokens at 5
-        self.burst_tokens = min(5, self.burst_tokens + int(elapsed * 3))
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)